src/Mailgun/Api/Pagination.php

```diff
@@ -22,12 +22,52 @@
     abstract protected function safeDeserialize(ResponseInterface $response, $className);
 
     /**
+     * @param mixed $response
+     *
+     * @return mixed|null
+     */
+    public function getPaginationNext($response)
+    {
+        return $this->getPaginationUrl($response->getNextUrl(), get_class($response));
+    }
+
+    /**
+     * @param mixed $response
+     *
+     * @return mixed|null
+     */
+    public function getPaginationPrevious($response)
```

Comment:
I would say `previous` or `previousPage` due we are already using a Pagination object does not make sense to "persist" using `pagination...`